hassio/src/dialogs/network/dialog-hassio-network.ts

```diff
@@ -292,13 +321,23 @@ export class DialogHassioNetwork extends LitElement implements HassDialog {
       | null
       | undefined
       | string[] = (ev.target as PaperInputElement).value;
+
```

Comment:
```suggestion
    const id = (ev.target as PaperInputElement).id;
	if (this._device.data[id] === value) {
		return;
	}
```

Reply:
Tried that first, does not work for the Array

Reply:
So store it as a string